FinalEngine.Editor.ViewModels/Inspectors/EntityComponentViewModel.cs

```diff
@@ -45,9 +58,18 @@ public EntityComponentViewModel(IEntityComponent component)
         this.propertyViewModels = new ObservableCollection<ObservableObject>();
 
         this.Name = component.GetType().Name;
+        this.IsVisible = true;
 
-        foreach (var property in component.GetType().GetProperties())
+        foreach (var property in component.GetType().GetProperties().OrderBy(x =>
```

Comment:
Determine whether I should abstract this functionality away into some service.

@KinsonDigital you mentioned this is something that I should do on discord but I'm not quite sure I understand why. Should this be abstracted away into a service that can retrieve properties of an `IEntityComponent`?